delete-kind.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Could we do:
```
#!/usr/bin/env bash
```

... for the script shebangs?

Reply:
Makes sense, changed it all.